### ISSUE TITLE
Server Crash When Placing Gasoline Engine

### ISSUE THREAD

**Issue by Patman0021** [ReikaKalseki/Reika_Mods_Issues#502]
After (unfortunately) removing Chromaticraft, when I try placing a gasoline engine, the server craps out with the attached crash report.

Is there anything I can do to fix this (other than dropping Galacticraft) ? I do have access to the hosted server.

[crash-2015-12-25_06.01.05-server.txt](https://github.com/ReikaKalseki/Reika_Mods_Issues/files/72044/crash-2015-12-25_06.01.05-server.txt)

**Comment by zemerick** [comment]
Not until they update with a fix.

**Comment by ljfa-ag** [comment]
Duplicate of #484
